packages/peregrine/lib/talons/MiniCart/useMiniCart.js

```diff
@@ -68,18 +74,12 @@ export const useMiniCart = props => {
 
     const handleRemoveItem = useCallback(
         async id => {
-            try {
-                await removeItem({
-                    variables: {
-                        cartId,
-                        itemId: id
-                    }
-                });
-            } catch (err) {
-                if (process.env.NODE_ENV !== 'production') {
-                    console.error('Cart Item Removal Error', err);
+            await removeItem({
```

Comment:
Even though our ErrorLink is awesome and auto logs, we still have to handle these mutation rejections. I have just been doing `catch { return; }`, but your comment might be a better solution. I think I've heard feedback that an empty return gives the impression there's a truthy response to expect, which this function does not have.

Reply:
As you can see I added half of these back. Will add back the ones I missed now :)

Reply:
Resolved in 9e88440